multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/model/SupportedParameters.java

```diff
@@ -137,13 +139,14 @@ public class SupportedParameters {
     public static final String FILTER = "filter";
     public static final String VISIBILITY = "visibility";
     // Configuration reference (old syntax):
-    public static final String MTA_VERSION = "mta-version";
-    public static final String MTA_ID = "mta-id";
-    public static final String MTA_MODULE = "mta-module";
-    public static final String MTA_PROVIDES_DEPENDENCY = "mta-provides-dependency";
-
-    public static final Set<String> CONFIGURATION_REFERENCE_PARAMETERS = Set.of(PROVIDER_NID, PROVIDER_ID, TARGET, VERSION, MTA_ID,
-                                                                                MTA_VERSION, MTA_PROVIDES_DEPENDENCY, NAMESPACE);
+    public static final String DEPRECATED_CONFIG_MTA_VERSION = "mta-version";
+    public static final String DEPRECATED_CONFIG_MTA_ID = "mta-id";
```

Comment:
Why "deprecated"? The values are the same

Reply:
The whole section `Configuration reference (old syntax)` will be removed soon and it will be more clear what to be removed as a whole bunch